### PR TITLE
Prerelease versions

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/512

The Preferences page was really outdated so I had to update that page to reflect the current state before making the addendum on prerelease versions.

### CODE REVIEW

File: docs/reference-guides/user-settings/user-preferences.md [hunk 1]
```diff
@@ -2,16 +2,59 @@
...
 Choose your background color for the Rancher UI. If you choose **Auto**, the background color changes from light to dark at 6 PM, and then changes back at 6 AM.
 
-## My Account
+## Login Landing Page
+
+Choose the page displayed after logging in. Options include:
+
+- The home page
+- The area you last visited
+- A specific cluster of your choosing
```
Comment (martyav) [inline docs/reference-guides/user-settings/user-preferences.md:27]:
```suggestion

- The home page.
- The area you last visited.
- A specific cluster of your choosing.
```

File: docs/reference-guides/user-settings/user-preferences.md [hunk 2]
```diff
@@ -2,16 +2,59 @@
...
+## Display Settings
+
+Choose the way certain information is displayed. The following can be configured:
+
+- Date format
+- Time format
+- Table rows per page
+- Number of clusters to show in side menu
+
+## Confirmation Setting
```
Comment (martyav) [inline docs/reference-guides/user-settings/user-preferences.md:38]:
```suggestion
## Confirmation Settings
```
Only one so far, but the plural matches the other headings and there might be more in the future

File: docs/reference-guides/user-settings/user-preferences.md [hunk 3]
```diff
@@ -2,16 +2,59 @@
...
+
+Choose whether to ask for confirmation when scaling down node pools.
+
+## Advanced Features
+
+- Enable "View in API"
+- Show system Namespaces managed by Rancher (not intended for editing or deletion)
+- Enable Dark/Light Theme keyboard shortcut toggle (shift + T)
+- Hide All Type Descriptions
+- Enable Extension developer features
```
Comment (martyav) [inline docs/reference-guides/user-settings/user-preferences.md:50]:
```suggestion
- Enable the "View in API" button.
- Show system Namespaces managed by Rancher (not intended for editing or deleting).
- Enable the Dark/Light Theme keyboard shortcut toggle (shift + T).
- Hide all Type descriptions.
- Enable Extension developer features.
```

Is the capitalization on all of these correct?

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/reference-guides/user-settings/user-preferences.md (hunk 3 above)
Comment (martyav) [reply docs/reference-guides/user-settings/user-preferences.md:50]: Per discussion, the capitalization and other problems are actually present in the Rancher UI

Comment (btat) [reply docs/reference-guides/user-settings/user-preferences.md:50]: Added a screenshot as discussed so that this is apparent to users.

**Review by martyav — APPROVED** [review]

**Pull request merged**